### PR TITLE
existency's easy morse code, code_golf

### PR DESCRIPTION
**Pull request by Existency**
Edit: 
Devia ter posto uma description...
Desafio 5 em code_golf para python e perl

### CODE REVIEW

File: 5_smooshed_morse_code/easy/code_golf/existency/python/Readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - Não há verificação de nenhum tipo de erro o que inclui mas não está limitado a caracteres inválidos e input nulo.
+    - Aceitar apenas uma palavra de cada vez. Aceitar múltiplas palavras iria pôr o código à volta de 130,135 chars.
+
+
+Por fim utilizar o atributo __file__ para poupar chars.
+Isto permite-me poupar (178-121) 57 chars.
+
+
+Usage:
+	python ' etianmsurwdkgohvf l pjbxcyzq  54 3   2       16       7   8 90' TypeYourWordHere
```
Comment (herulume) [inline 5_smooshed_morse_code/easy/code_golf/existency/python/Readme.md]: This won't do. It's clever, and props for that, but you are passing "part of the program" when calling it.

**Review by herulume — CHANGES_REQUESTED** [review]
Your file names are messed up.

Can you confirm me that Text::Morse is not an external dependency that we need to install? 

You are passing the input using argv, you need to pass it in the stdin. This wasn't clear and it's our fault, sorry.

File: 5_smooshed_morse_code/easy/code_golf/existency/python/Readme.md (hunk 1 above)
Comment (Existency) [reply 5_smooshed_morse_code/easy/code_golf/existency/python/Readme.md]: Text::Morse is a dependency, will change things around in perl

I am using the same "trick", passing the name of the file to the program to save some characters. 
In perl it is 
`$0`
And python
`__file__`

I really thought we could pass it through argv, will switch both programs to stdin and fix the aforementioned issues in my code.

Comment (Existency) [reply 5_smooshed_morse_code/easy/code_golf/existency/python/Readme.md]: The problems have been fixed. 

By request of herulume, tagging @BrunoMartins11

**Pull request merged**